tests/conditional_compilation/test_infer.py

```diff
@@ -4,19 +4,31 @@
 
 """ Test inference with conditional compiled binaries.
 """
-import sys
+import sys, os
 from proc_utils import cmd_exec  # pylint: disable=import-error
 from install_pkg import get_openvino_environment  # pylint: disable=import-error
 
 
-def test_infer(test_id, model, artifacts):
+def test_infer(test_id, model, artifacts, infer_tool,  install_dir, install_cc_dir):
     """ Test inference with conditional compiled binaries
     """
-    install_prefix = artifacts / test_id / "install_pkg"
-    exe_suffix = ".exe" if sys.platform == "win32" else ""
-    benchmark_app = install_prefix / "bin" / f"benchmark_app{exe_suffix}"
-    returncode, _ = cmd_exec(
-        [str(benchmark_app), "-d=CPU", f"-m={model}", "-niter=1", "-nireq=1"],
-        env=get_openvino_environment(install_prefix),
+    tmp = os.environ["PATH"]
+    tmp = tmp.replace(os.path.join(*list(install_cc_dir.parts)), os.path.join(*list(install_dir.parts)))
+    os.environ["PATH"] = tmp
+    model_result_path = os.path.join(*list(artifacts.parts), model.stem)
+    returncode, output = cmd_exec(
```

Comment:
please, move inference of original OVINO to a fixture (as example see `install_cc_dir`). In this case it will be called once